src/modules/add-licences/controller.js

```diff
@@ -350,26 +407,26 @@ async function postAddressSelect (request, h) {
     const verification = await CRM.createVerification(entityId, companyEntityId, selectedIds);
 
     // Get the licence containing the selected verification address
-    const addressLicence = await getLicence(address);
+    const addressLicence = await getLicence(selectedAddressId);
 
     // Post letter
-    await Notify.sendSecurityCode(addressLicence, verification.verification_code);
+    await Notify.sendSecurityCode(addressLicence, fao, verification.verification_code);
 
     // Delete data in session
     request.sessionStore.delete('addLicenceFlow');
-    //
+
     // add the company id to the cookie to configure company switcher correctly
     loginHelpers.selectCompany(request, { entityId: companyEntityId, name: companyName });
 
-    const viewContext = await getAddressSelectViewContext(request, verification, addressLicence);
+    const viewContext = await getAddressSelectViewContext(request, verification, addressLicence, fao);
 
-    return h.view('water/licences-add/verification-sent', viewContext);
-  } catch (err) {
-    if (err.name === 'InvalidAddressError') {
-      return h.redirect('/select-address?error=invalidAddress');
-    }
-    throw err;
+    return h.view('nunjucks/licences-add/verification-sent.njk', viewContext, { layout: false });
```

Comment:
For the new and improved Nunjucks views, if we could make the folder name match the module name